src/main.c

```diff
@@ -87,7 +87,7 @@
 #  define _POSIX_SOURCE 1               /* Solaris needs this */
 #endif
 
-extern char origbotname[], userfile[], botnetnick[];
+extern char origbotname[], userfile[121], botnetnick[];
```

Comment:
Adding another magic number to fix another?
Probably a better idea to move `userfile` from `users.c` to `userrec.c` and `backup_userfile()` from `main.c` to `userrec.c` as well and getting rid of some of these externs and magic numbers. Not really the scope of this PR though.

Reply:
Documented the magic number source, we can clean it up later if we think the benefit outweighs the risk of breakage